etc/ryu/faucet/faucet.yaml

```diff
@@ -25,6 +25,7 @@ routers:
 dps:
     sw1:
         dp_id: 0x1
+        dp_name: 'sw1'
```

Comment:
Overriding dp_name is a "faucet expert" level feature that we don't recommend new users use due to some possible side-effects of the feature that you must understand before using this feature. As such we don't think this should be shown in the example configuration, please see PR#1107 for more information.